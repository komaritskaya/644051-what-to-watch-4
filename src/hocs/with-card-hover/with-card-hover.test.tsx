import React from 'react';
import renderer from 'react-test-renderer';

import withCardHover from './with-card-hover';

interface MockComponentProps {
  children: React.ReactNode;
}

const MockComponent = (props: MockComponentProps) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const MockComponentWrapped = withCardHover(MockComponent);

it(`withCardHover is rendered correctly`, () => {
  const tree = renderer.create(
      <MockComponentWrapped
        isPlaying={true}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
