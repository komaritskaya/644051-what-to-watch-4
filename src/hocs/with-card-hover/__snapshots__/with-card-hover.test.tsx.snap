// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withCardHover is rendered correctly 1`] = `<div />`;
